Sources/BraintreePayPal/BTPayPalDriver.m

```diff
@@ -94,27 +94,22 @@ - (void)dealloc {
 
 #pragma mark - Billing Agreement
 
-- (void)requestBillingAgreement:(BTPayPalRequest *)request
+- (void)requestBillingAgreement:(BTPayPalVaultRequest *)request
                      completion:(void (^)(BTPayPalAccountNonce *tokenizedCheckout, NSError *error))completionBlock {
-    [self requestPayPalCheckout:request
-             isBillingAgreement:YES
-                     completion:completionBlock];
+    [self requestPayPalCheckout:request completion:completionBlock];
 }
 
 #pragma mark - Express Checkout (One-Time Payments)
 
 - (void)requestOneTimePayment:(BTPayPalCheckoutRequest *)request
                    completion:(void (^)(BTPayPalAccountNonce *tokenizedCheckout, NSError *error))completionBlock {
-    [self requestPayPalCheckout:request
-             isBillingAgreement:NO
-                     completion:completionBlock];
+    [self requestPayPalCheckout:request completion:completionBlock];
 }
 
 #pragma mark - Helpers
 
 /// A "Hermes checkout" is used by both Billing Agreements (Vault) and One-Time Payments (Checkout)
```

Comment:
```suggestion
/// Used by both Billing Agreements (Vault) and One-Time Payments (Checkout)
```

Not sure if we need to mention `Hermes checkout` here specifically, just documenting how this is used should be fine

Reply:
Yeah, this comment isn't super helpful. In the next PR, this method is made public and renamed to `tokenizePayPalAccount`, and we delete this comment alongside that change. So I'm just going to leave it as-is here, knowing that it'll be deleted when the next PR is merged.